lib/_http_outgoing.js

```diff
@@ -275,19 +284,13 @@ function _writeRaw(data, encoding, callback) {
     return conn.write(data, encoding, callback);
   }
   // Buffer, as long as we're not destroyed.
-  return this._buffer(data, encoding, callback);
-}
-
-
-OutgoingMessage.prototype._buffer = function _buffer(data, encoding, callback) {
   this.output.push(data);
   this.outputEncodings.push(encoding);
   this.outputCallbacks.push(callback);
   this.outputSize += data.length;
-  if (typeof this._onPendingData === 'function')
-    this._onPendingData(data.length);
+  this._onPendingData(data.length);
   return false;
-};
+}
```

Comment:
Isn't this the end of `OutgoingMessage.prototype._send = function _send`, so should have `;`